doc/releases/release-notes-3.7.rst

```diff
@@ -120,6 +120,17 @@ Drivers and Sensors
 
 * Entropy
 
+* eSPI
+
+  * The macros ``ESPI_SLAVE_TO_MASTER`` and ``ESPI_MASTER_TO_SLAVE`` were renamed to
+    ``ESPI_TARGET_TO_CONTROLLER`` and ``ESPI_CONTROLLER_TO_TARGET`` respectively to reflect
+    new terminology in eSPI 1.5 specification.
+  * The following virtual enum values ``ESPI_VWIRE_SIGNAL_SLV_BOOT_STS``,
+    ``ESPI_VWIRE_SIGNAL_SLV_BOOT_DONE`` and ``ESPI_VWIRE_SIGNAL_SLV_GPIO_<NUMBER>`` were renamed to
+    ``ESPI_VWIRE_SIGNAL_TARGET_BOOT_STS``, ``ESPI_VWIRE_SIGNAL_TARGET_BOOT_DONE`` and
+    ``ESPI_VWIRE_SIGNAL_TARGET_GPIO_<NUMBER>`` respectively to reflect new terminology
+    in eSPI 1.5 specification.
```

Comment:
this shouldn't be paraphrasing the migration guide entry. Either drop altogether or, probably better, just make it a simpler bullet point "Renamed several eSPI APIs to adopt terminology from version 1.5 of the specification" or something like this?

Reply:
Summarized